pkgs/by-name/ak/akkoma-admin-fe/package.nix

```
@@ -39,6 +40,7 @@ stdenv.mkDerivation (finalAttrs: {
     nodejs
     pkg-config
     python3
+    python3Packages.distutils
```

Why not use withPackages?
```suggestion
    (python3.withPackages (p: [p.distutils]))
```
This seems to be integrated with Python tighter